docs/source/blog/arc-collaboration.md

```diff
@@ -0,0 +1,90 @@
+
+This work was part of a collaboration involving Sepiedeh Keshavarzi (Cambridge University), ARC (UCL), and the core `movement` development team and has been funded by the Wellcome Trust.
+
+*At a glance*
+- **Regions of Interest**: An RoI backend was added for analysis of RoIs, allowing future development of the napari plugin.
+- **Plot Wrapper Functions**: Added to aid in initial "eyeballing" of data, making it easy to create common analysis plots with a single line of code.
+- **New examples**: [Extend your analysis methods along data dimensions](https://movement.neuroinformatics.dev/examples/broadcasting_your_own_methods.html#sphx-glr-examples-broadcasting-your-own-methods-py), [Pupil tracking](https://movement.neuroinformatics.dev/examples/mouse_eye_movements.html).
+## Original Roadmap
+
+Between January and February 2025, the `movement` developer team decided to focus on expanding the suite of tools useful for the analysis of spatial navigation.
+This work started with the creation of so-called "backend" functions that can perform general operations on vectors in an efficient manner, from which friendlier functions could be provided.
+Once these features were in place, development would start on supporting "Regions of Interest" (RoIs) within `movement`.
+RoIs are essentially labelled regions in space that have some significance in the experimental setup - the location of a nest, or the extent of the enclosure, for example.
+By providing a way to store and describe these objects within `movement`, analysis workflows can be simplified and can make use of the existing "backend" functions to provide convenient access to interesting quantities; such as the distance of each individual from a RoI, or the relative orientation of an individual from the nearest wall of the enclosure.
+
+There were also a few independent bursts of development in other areas; including providing sample pupillometry data and an example workflow showing how `movement` can be used to explore different aspects of the data (e.g. position, velocity, and diameter of the pupil), and how to normalise and filter the data.
+
+A copy of the original roadmap that was [shared on Zulip](https://neuroinformatics.zulipchat.com/#narrow/channel/406001-Movement/topic/Roadmap.3A.20Spatial.20Navigation/near/495022291) is provided below.
+
+![Original feature roadmap for Jan-Feb.](../_static/blog_posts/roadmap-jan-feb-2025.png)
+
+## What's Been Introduced?
+
+In addition to the topic listed below, we have also introduced the following smaller, self-contained features to `movement`:
+
+- We have new sample datasets available, including two eye-movement tracking (pupillometry) datasets. An example of how to analyse this data using `movement` was introduced in [#429](https://github.com/neuroinformatics-unit/movement/pull/429) and can be found [here](https://movement.neuroinformatics.dev/examples/mouse_eye_movements.html#sphx-glr-examples-mouse-eye-movements-py).
```

Comment:
Again consider using the specific cross-regerencing syntax for examples:

https://sphinx-gallery.github.io/stable/advanced.html#cross-referencing